examples/javascript/uart.js

```diff
@@ -22,33 +22,33 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-var m = require('mraa'); //require mraa
-console.log('MRAA Version: ' + m.getVersion());
-u = new m.Uart(0)
+const mraa = require('mraa'); //require mraa
+console.log('MRAA Version: ' + mraa.getVersion());
+let uart = new mraa.Uart(0)
```

Comment:
Let's add semicolons to all lines in this file to align with the style proposed in this PR